modules/core/src/test/java/org/apache/ignite/internal/processors/security/cache/EntryProcessorPermissionCheckTest.java

```diff
@@ -79,7 +79,7 @@ private void runOperation(Ignite verifierNode, BiConsumer<String, T2<String, Int
 
         c.accept(CACHE_NAME, entry);
 
-        assertThat(verifierNode.cache(CACHE_NAME).get(entry.getKey()), is(entry.getValue()));
+        assertThat((Integer)(verifierNode.cache(CACHE_NAME).get(entry.getKey())), is(entry.getValue()));
```

Comment:
I think we should not cast here, but parametrize cache directly:
`verifierNode.<String, Integer>cache(CACHE_NAME).get(entry.getKey())`

Reply:
ok